lifecycle-operator/main.go

```diff
@@ -283,7 +282,6 @@ func main() {
 		mgr.GetClient(),
 		workloadVersionEventSender,
 		workloadVersionLogger,
-		noop.NewTracerProvider().Tracer("keptn/lifecycle-operator/workloadversion"),
```

Comment:
could we also use `telemetry.GetOtelInstance().GetTracer()` here and keep the tracer as a property of the `EvaluationHandler`? This way the number of arguments for the `EvaluationHandler` methods can be reduced since we don't have the tracer across its different methods (I think this is also how we do it in the `TaskHandler`)

Reply:
at this point I would probably remove the evaluation handler entirely from here and initialize it internally like we do for the task handler...

Reply:
do it